static/schema.json

```diff
@@ -745,7 +745,41 @@
         }
       }
     },
-    "Native_query_sql": {
+    "NativeQuerySql": {
```

Comment:
This feels like it's related to @soupi 's recent `.sql` files change for native queries rather than domain types?